rero_ils/modules/libraries/mappings/v7/libraries/library-v0.0.1.json

```diff
@@ -157,6 +160,9 @@
               "phone": {
                 "type": "text"
               },
+              "extra": {
```

Comment:
I think the field `extra`  is added `twice` in this file

Reply:
Yes : one for billing address, one for shipping address. I tried to use a "definition" for both address but as specifications is not perfectly the same I didn't success.